x/group/module/abci_test.go

```diff
@@ -0,0 +1,233 @@
+package module_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/simapp"
+	"github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/cosmos/cosmos-sdk/x/group"
+	"github.com/cosmos/cosmos-sdk/x/group/module"
+	"github.com/stretchr/testify/require"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+)
+
+func TestProposalsByVPEnd(t *testing.T) {
```

Comment:
what about just using this naming?
```suggestion
func TestEndBlocker(t *testing.T) {
```

`ProposalsByVPEnd` sounds a bit too restrictive.